dist/css/main.css

```diff
@@ -17693,6 +17693,14 @@ body.theme-accent-white .main-nav .dropdown.current .navbar-touch-caret:hover, b
   background-color: #FFF;
 }
 
+.main-nav .dropdown-menu .dropdown-item {
+  padding: 0 1.5rem;
+}
+
+.main-nav .dropdown-menu .dropdown-item:not(.active):hover {
+  background-color: #f5f5f5;
```

Comment:
Is this always the same, even with editable themes?

Reply:
Yep, I remember having a discussion with Paul about this, he said that this colour remains the same despite the background colour changes :)